code/09-conclusion-challenge-2/Makefile

```diff
@@ -10,7 +10,7 @@ ZIPF_ARCHIVE=$(ZIPF_DIR).tar.gz
 .PHONY : all
 all : $(ZIPF_ARCHIVE)
 
-$(ZIPF_ARCHIVE) : $(ZIPF_DIR)
+$(ZIPF_ARCHIVE) : $(ZIPF_DIR) $(PNG_FILES) results.txt
```

Comment:
Thanks @gcapes!  This is definitely the better way to do it, so I've made this change.